integration-tests/api-gateway/abandon/abandon-happy.test.ts

```diff
@@ -19,23 +21,40 @@ describe("Given the session is valid and expecting to abandon the journey", () =
   let sessionTableName: string;
   let state: string;
   let personIDTableName: string;
+  let audience: string | undefined;
+  let issuer: string | undefined;
   let output: Partial<{
     CommonStackName: string;
     StackName: string;
     NinoUsersTable: string;
     UserAttemptsTable: string;
+    PrivateApiGatewayId: string;
   }>;
 
-  beforeEach(async () => {
+  const clientId = "ipv-core-stub-aws-headless";
+
+  beforeAll(async () => {
     output = await stackOutputs(process.env.STACK_NAME);
+    const commonStack = output.CommonStackName;
     sessionTableName = `session-${output.CommonStackName}`;
 
-    const session = await createSession();
+    [audience, issuer] = await getSSMParameters(
+      `/${commonStack}/clients/${clientId}/jwtAuthentication/audience`,
+      `/${commonStack}/clients/${clientId}/jwtAuthentication/issuer`
+    );
+  });
+
+  beforeEach(async () => {
+    const data = await getJarAuthorization(clientId, audience, issuer);
```

Comment:
Do we definitely need to override client Id and issuer? Looks like we're using the default values the /start endpoint would use anyway?

Reply:
Think `aud` is required by the stub though @jkunle ?